apps/console/src/lib/subscription-plan/plans.ts

```diff
@@ -0,0 +1,78 @@
+import { FeatureEnum } from './feature-enum'
+import { PlanEnum } from '@/lib/subscription-plan/plan-enum.ts'
+
+export const featureUtil = {
+  notAvailableText: 'Not available in your current plan',
+  notAvailableFullText: 'Forbidden. This feature is not available in your current subscription plan. Please upgrade your plan from your billing settings to access this feature.',
+  baseModule: () => [
+    FeatureEnum.ASSETS,
+    FeatureEnum.CONTROL_REPORT,
+    FeatureEnum.CONTROLS,
+    FeatureEnum.DASHBOARD,
+    FeatureEnum.ENTITIES,
+    FeatureEnum.EVIDENCE,
+    FeatureEnum.GROUPS,
+    FeatureEnum.ONBOARDING,
+    FeatureEnum.ORGANIZATION,
+    FeatureEnum.ORGANIZATION_SETTINGS,
+    FeatureEnum.POLICIES,
+    FeatureEnum.PROCEDURES,
+    FeatureEnum.PROGRAMS,
+    FeatureEnum.QUESTIONNAIRES,
+    FeatureEnum.REPORTING,
+    FeatureEnum.RISKS,
+    FeatureEnum.STANDARDS,
+    FeatureEnum.SUBSCRIPTION,
+    FeatureEnum.TASKS,
+    FeatureEnum.USER_SETTINGS,
```

Comment:
```suggestion
    FeatureEnum.DASHBOARD,
    FeatureEnum.GROUPS,
    FeatureEnum.ONBOARDING,
    FeatureEnum.ORGANIZATION,
    FeatureEnum.ORGANIZATION_SETTINGS,
    FeatureEnum.SUBSCRIPTION,
    FeatureEnum.TASKS,
    FeatureEnum.USER_SETTINGS,
```

Only these are in the base module